apps/condo/domains/acquiring/utils/freezeBillingReceipt.js

```diff
@@ -0,0 +1,31 @@
+const { getById } = require('@core/keystone/schema')
+
+// TODO(savelevMatthew): Replace with single request from serverSchema after gql refactoring
+/**
+ * Combine multiple billing entities into single object
+ *
+ * @param {Object} flatReceipt BillingReceipt received by "find" from "@core/keystone/schema"
+ */
+
+async function freezeBillingReceipt (flatReceipt) {
```

Comment:
Why the name contains `freeze…` if in comment it states "Combine"?

Reply:
Because it's freezing state of models at the current time to allow support resolving possible conflicts....but I agree, I'll write more complex description

Reply:
changed description 👍